### PR TITLE
Remove stale Roslyn feature branches

### PR DESCRIPTION
**Pull request by shyamnamboodiripad**

### CODE REVIEW

**Comment by shyamnamboodiripad** [comment]
@jaredpar @jasonmalinowski

**Comment by shyamnamboodiripad** [comment]
@rchande for the "ins" branch

File: jobs/data/repolist.txt [hunk 1]
```diff
@@ -71,15 +71,12 @@ dotnet/roslyn branch=dev16 server=dotnet-ci
 dotnet/roslyn branch=master server=dotnet-ci
 dotnet/roslyn branch=master subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
 dotnet/roslyn branch=microupdate server=dotnet-ci
-dotnet/roslyn branch=features/async-main server=dotnet-ci
 dotnet/roslyn branch=features/DefaultInterfaceImplementation server=dotnet-ci
 dotnet/roslyn branch=features/editorconfig-in-compiler server=dotnet-ci
-dotnet/roslyn branch=features/ins server=dotnet-ci
```
Comment (jasonmalinowski) [inline jobs/data/repolist.txt]: @rchande what's the status of this branch?

Comment (shyamnamboodiripad) [inline jobs/data/repolist.txt]: Ravi confirmed that this is ok to delete.

**Pull request merged**